### ISSUE TITLE
Make the package installable via pip

### ISSUE THREAD

**Issue by d-krupke** [CG-SHOP/pyutils23#2]
Currently, you need to install the package by hand because of the native dependencies. We could make it installable via pip using either `cibuildwheels` or `sdists`.

* `cibuildwheels`: May have problems with conda and the compilation of CGAL will be to expensive to use GitHubs free CI.
* `sdists`: Seems easier but we would need to find a new way to trigger conan before build. This may be possible by driving conan from cmake (but I am not sure, if the cmake will not become unusable or at least ugly when also used for building pure C++-libraries).

**Comment by d-krupke** [comment]
An alternative is to also build and upload the package by hand for every update. However, there may still be problems with `conda`, as conda ships its own glibc, that can be incompatible to the one of the system.

The best option seems to me creating a `sdist`-package on PyPI and finding an elegant way to drive conan.

**Comment by d-krupke** [comment]
So I played around a little and we could simple trigger conan from cmake whenever `SKBUILD` is defined. This gets me much further, but for some reason not all dependencies will be installed (however, cgal will).

```
if(SKBUILD)
    execute_process(COMMAND "conan install . --build=missing -if cmake")
endif()
```

**Comment by d-krupke** [comment]
```
if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
    message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
    file(DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/0.18.1/conan.cmake"
            "${CMAKE_BINARY_DIR}/conan.cmake"
            TLS_VERIFY ON)
endif()

include(${CMAKE_BINARY_DIR}/conan.cmake)

conan_cmake_configure()

conan_cmake_autodetect(settings)

conan_cmake_install(PATH_OR_REFERENCE .
        BUILD missing
        REMOTE conancenter
        SETTINGS ${settings})
```
Seems to work with fallback mode, producing  a lot of errors/warnings that could confuse the user.

**Comment by d-krupke** [comment]
Seems to work on the development branch by doing an ugly hack and adding a pyproject.toml. Needs further verification.

**Comment by d-krupke** [comment]
Should be fixed with v0.2.0
